changelog/@unreleased/pr-15.v2.yml

```diff
@@ -0,0 +1,15 @@
+type: improvement
+improvement:
+  description: |-
+    Add isEmpty String error prone check
+
+    ## Before this PR
+    isEqualTo("") and isNotEqualTo("") are allowed
+
+    ## After this PR
+    They are changed to isEmpty() and isNotEmpty()
+
+    ## Possible downsides?
```

Comment:
Normally just the 'after' piece is sufficient. For this change, I think I'd use "AssertjRefactoring rewrites assertThat(string).isEqualTo("") as assertThat(string).isEmpty()"